src/bokeh/command/subcommands/settings.py

```diff
@@ -99,6 +86,38 @@
     'Settings',
 )
 
+#-----------------------------------------------------------------------------
+# Helpers
+#-----------------------------------------------------------------------------
```

Comment:
We don't have a helpers banner in any of our other modules, normally a function like this would go under the "Dev API" banner that all modules have